src/CardWrapper/CardWrapper.js

```diff
@@ -1,47 +1,66 @@
 import React from 'react';
 import './CardWrapper.css';
-import Card from '../Card/Card.js'
+// import Card from '../Card/Card.js'
 import VehicleCard from '../VehicleCard/VehicleCard.js'
 import PlanetCard from '../PlanetCard/PlanetCard.js'
+import PeopleCard from '../PeopleCard/PeopleCard.js'
 
 
+const fetchPlanetApi = (obj) => {
+  let planetObject = {}
+  let { name } = obj;
+  fetch(obj.homeworld)
```

Comment:
If this API call is for a particular planet I would move it into a PlanetCard component. Having trouble figuring out if these are the the files you're having trouble with. You've commented out the imported files at the top here so at this point things should break on line 49.